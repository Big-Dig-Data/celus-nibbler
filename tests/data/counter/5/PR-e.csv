Report_Name,Platform Master Report,,,,,,,,,,,,,,,
Report_ID,PR,,,,,,,,,,,,,,,
Release,5,,,,,,,,,,,,,,,
Institution_Name,Celus University,,,,,,,,,,,,,,,
Institution_ID,,,,,,,,,,,,,,,,
Metric_Types,,,,,,,,,,,,,,,,
Report_Filters,Data_Type=Book|Database|Journal|Platform; Access_Method=Regular,,,,,,,,,,,,,,,
Report_Attributes,Attributes_To_Show=Data_Type|Access_Method,,,,,,,,,,,,,,,
Exceptions,,,,,,,,,,,,,,,,
Reporting_Period,Begin_Date=2020-01-01; End_Date=2020-12-31,,,,,,,,,,,,,,,
Created,2022-01-01T00:00:00Z,,,,,,,,,,,,,,,
Created_By,Ovid,,,,,,,,,,,,,,,

Platform,Data_Type,Access_Method,Metric_Type,Reporting_Period_Total,Jan-2020,Feb-2020,Mar-2020,Apr-2020,May-2020,Jun-2020,Jul-2020,Aug-2020,Sep-2020,Oct-2020,Nov-2020,Dec-2020
Platform1,Journal,Regular,Total_Item_Investigations,234,0,22,4,25,6,7,23,5,30,56,30,26
Platform1,Journal,Regular,Total_Item_Requests,205,0,18,4,22,5,6,22,5,23,52,27,21
Platform1,Journal,Regular,Unique_Item_Investigations,202,0,17,4,20,6,5,22,4,29,50,26,19
Platform1,Journal,Regular,Unique_Item_Requests,174,0,13,4,17,5,4,21,4,22,46,23,15
Platform2,Journal,Regular,Total_Item_Investigations,253,0,74,38,102,19,0,4,9,7,0,0,0
Platform2,Journal,Regular,Total_Item_Requests,244,0,71,37,100,17,0,4,9,6,0,0,0
Platform2,Journal,Regular,Unique_Item_Investigations,140,0,37,20,58,14,0,2,4,5,0,0,0
Platform2,Journal,Regular,Unique_Item_Requests,131,0,34,19,56,12,0,2,4,4,0,0,0
