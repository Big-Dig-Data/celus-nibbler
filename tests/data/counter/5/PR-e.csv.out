"2020-01-01","2020-01-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Investigations","0"
"2020-02-01","2020-02-29","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Investigations","22"
"2020-03-01","2020-03-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Investigations","4"
"2020-04-01","2020-04-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Investigations","25"
"2020-05-01","2020-05-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Investigations","6"
"2020-06-01","2020-06-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Investigations","7"
"2020-07-01","2020-07-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Investigations","23"
"2020-08-01","2020-08-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Investigations","5"
"2020-09-01","2020-09-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Investigations","30"
"2020-10-01","2020-10-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Investigations","56"
"2020-11-01","2020-11-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Investigations","30"
"2020-12-01","2020-12-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Investigations","26"
"2020-01-01","2020-01-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Requests","0"
"2020-02-01","2020-02-29","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Requests","18"
"2020-03-01","2020-03-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Requests","4"
"2020-04-01","2020-04-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Requests","22"
"2020-05-01","2020-05-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Requests","5"
"2020-06-01","2020-06-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Requests","6"
"2020-07-01","2020-07-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Requests","22"
"2020-08-01","2020-08-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Requests","5"
"2020-09-01","2020-09-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Requests","23"
"2020-10-01","2020-10-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Requests","52"
"2020-11-01","2020-11-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Requests","27"
"2020-12-01","2020-12-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Total_Item_Requests","21"
"2020-01-01","2020-01-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Investigations","0"
"2020-02-01","2020-02-29","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Investigations","17"
"2020-03-01","2020-03-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Investigations","4"
"2020-04-01","2020-04-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Investigations","20"
"2020-05-01","2020-05-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Investigations","6"
"2020-06-01","2020-06-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Investigations","5"
"2020-07-01","2020-07-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Investigations","22"
"2020-08-01","2020-08-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Investigations","4"
"2020-09-01","2020-09-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Investigations","29"
"2020-10-01","2020-10-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Investigations","50"
"2020-11-01","2020-11-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Investigations","26"
"2020-12-01","2020-12-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Investigations","19"
"2020-01-01","2020-01-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Requests","0"
"2020-02-01","2020-02-29","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Requests","13"
"2020-03-01","2020-03-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Requests","4"
"2020-04-01","2020-04-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Requests","17"
"2020-05-01","2020-05-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Requests","5"
"2020-06-01","2020-06-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Requests","4"
"2020-07-01","2020-07-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Requests","21"
"2020-08-01","2020-08-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Requests","4"
"2020-09-01","2020-09-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Requests","22"
"2020-10-01","2020-10-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Requests","46"
"2020-11-01","2020-11-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Requests","23"
"2020-12-01","2020-12-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform1","Unique_Item_Requests","15"
"2020-01-01","2020-01-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Investigations","0"
"2020-02-01","2020-02-29","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Investigations","74"
"2020-03-01","2020-03-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Investigations","38"
"2020-04-01","2020-04-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Investigations","102"
"2020-05-01","2020-05-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Investigations","19"
"2020-06-01","2020-06-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Investigations","0"
"2020-07-01","2020-07-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Investigations","4"
"2020-08-01","2020-08-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Investigations","9"
"2020-09-01","2020-09-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Investigations","7"
"2020-10-01","2020-10-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Investigations","0"
"2020-11-01","2020-11-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Investigations","0"
"2020-12-01","2020-12-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Investigations","0"
"2020-01-01","2020-01-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Requests","0"
"2020-02-01","2020-02-29","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Requests","71"
"2020-03-01","2020-03-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Requests","37"
"2020-04-01","2020-04-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Requests","100"
"2020-05-01","2020-05-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Requests","17"
"2020-06-01","2020-06-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Requests","0"
"2020-07-01","2020-07-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Requests","4"
"2020-08-01","2020-08-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Requests","9"
"2020-09-01","2020-09-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Requests","6"
"2020-10-01","2020-10-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Requests","0"
"2020-11-01","2020-11-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Requests","0"
"2020-12-01","2020-12-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Total_Item_Requests","0"
"2020-01-01","2020-01-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Investigations","0"
"2020-02-01","2020-02-29","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Investigations","37"
"2020-03-01","2020-03-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Investigations","20"
"2020-04-01","2020-04-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Investigations","58"
"2020-05-01","2020-05-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Investigations","14"
"2020-06-01","2020-06-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Investigations","0"
"2020-07-01","2020-07-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Investigations","2"
"2020-08-01","2020-08-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Investigations","4"
"2020-09-01","2020-09-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Investigations","5"
"2020-10-01","2020-10-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Investigations","0"
"2020-11-01","2020-11-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Investigations","0"
"2020-12-01","2020-12-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Investigations","0"
"2020-01-01","2020-01-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Requests","0"
"2020-02-01","2020-02-29","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Requests","34"
"2020-03-01","2020-03-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Requests","19"
"2020-04-01","2020-04-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Requests","56"
"2020-05-01","2020-05-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Requests","12"
"2020-06-01","2020-06-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Requests","0"
"2020-07-01","2020-07-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Requests","2"
"2020-08-01","2020-08-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Requests","4"
"2020-09-01","2020-09-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Requests","4"
"2020-10-01","2020-10-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Requests","0"
"2020-11-01","2020-11-30","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Requests","0"
"2020-12-01","2020-12-31","","","","","","Access_Method:Regular|Data_Type:Journal|Platform:Platform2","Unique_Item_Requests","0"
